### ISSUE TITLE
Compile error for android template

### ISSUE THREAD

**Issue by AlexHolly** [godotengine/godot#12266]
Godot Master
Kubuntu
It was working ~3 days ago.

Any Idea why I am getting this?

`scons platform=android target=release_debug bits=64 builtin_libpng=yes builtin_openssl=yes builtin_zlib=yes
`
```
In file included from modules/gdnative/pluginscript/pluginscript_instance.cpp:37:
modules/gdnative/pluginscript/pluginscript_script.h:34:10: fatal error: 'iostream' file not found
#include <iostream>
         ^~~~~~~~~~
1 error generated.
scons: *** [modules/gdnative/pluginscript/pluginscript_instance.android.opt.debug.64.armv7.neon.o] Error 1
scons: building terminated because of errors.
```

**Comment by akien-mga** [comment]
Bad bad @touilleMan using the STL in Godot...

**Comment by neikeq** [comment]
@touilleMan That's sacrilege.

**Comment by touilleMan** [comment]
![shame](https://pics.me.me/shame-shame-shame-memeful-com-14049487.png)
Sorry sorry sorry #12269

**Comment by akien-mga** [comment]
Fixed by #12269.
